core/templates/dev/head/components/state-editor/state-editor.directive.html

```diff
@@ -10,7 +10,7 @@
     </div>
   </div>
 </md-card>
-
+<button ng-click="fetchAd()">Check AD</button>
```

Comment:
I'm assuming this means "Check Answer Details". If so, please don't use abbreviations; just use the full term.

Reply:
Done